docs/installation/sources/sources_linux.rst

```diff
@@ -111,6 +110,56 @@ For example, on Ubuntu use the command:
 
    sudo apt install libssl-dev
 
+.. _libp11_sl:
+
+Libp11 and SoftHSM libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Libp11 provides PKCS#11 support for OpenSSL. This is an optional dependency,
+that is needed only when *eprosima Fast DDS* is used with security and PKCS#11 URIs.
+
+Install libp11_ using the package manager of the appropriate Linux distribution.
+For example, on Ubuntu use the command:
+
+.. code-block:: bash
+
+   sudo apt install libp11-dev libengine-pkcs11-openssl
+
+SoftHSM is a software implementation of an HSM (Hardware Security Module).
+If *eProsima Fast DDS* tests are activated and *libp11* is installed
+on the system, SoftHSM is additionally required to run tests of PKCS#11 features.
+
+Install SoftHSM_ using the package manager of the appropriate Linux distribution.
+For example, on Ubuntu use the command:
+
+.. code-block:: bash
+
+   sudo apt install softhsm2
+
+OpenSSL access HSM and other hardware devices through its engine functionality.
+In order to set up a new engine the OpenSSL configuration files (usually `/etc/ssl/openssl.cnf`)
+must be updated specifying the libp11_ and hardware module (here SoftHSM_) dynamic libraries
+location.
+
+This configuration step can be avoided using p11kit_ which allows OpenSSL to find PKCS#11
+devices on runtime without static configuration. This kit is often available through
+the distro package manager. On Ubuntu **apt**, for example: 
```

Comment:
```suggestion
the Linux distribution package manager. On Ubuntu, for example:
```

distro is not a valid word according with the spelling test.

`apt` is not mentioned in the rest of the file.